ratelimits/bucket.go

```diff
@@ -76,8 +76,9 @@ func newRegIdDomainBucketKey(name Name, regId int64, orderName string) (string,
 
 // newFQDNSetBucketKey validates and returns a bucketKey for limits that use the
 // 'enum:fqdnSet' bucket key format.
-func newFQDNSetBucketKey(name Name, orderNames []string) (string, error) {
-	id := string(core.HashNames(orderNames))
+func newFQDNSetBucketKey(name Name, orderNames []string) (string, error) { //nolint: unparam
+	hash := core.HashNames(orderNames)
+	id := fmt.Sprintf("%x", hash)
 	err := validateIdForName(name, id)
```

Comment:
Why not do the validate before computing the hash, so that the new regexp in names.go isn't necessary?

Reply:
Ah, fair enough.